doc/migration/v0.1.0-migration-guide.md

```diff
@@ -0,0 +1,278 @@
+# Migration Guide from v0.0.x to v0.1.0
+
+This document describes how to migrate an operator project built using Operator SDK `v0.0.x` to the project structure required by `v0.1.0`. 
+
+The recommended way to migrate your project is to initialize a new `v0.1.0` project, then copy your code into the new project and modify as described below.
+
+This guide goes over migrating the memcached-operator, an example project from the user guide, to illustrate migration steps. See the [v0.0.7 memcached-operator][v0.0.7-memcached-operator] and [v0.1.0 memcached-operator][v0.1.0-memcached-operator] project structures for pre- and post-migration examples, respectively.
+
+## Create a new v0.1.0 project
+
+Rename your `v0.0.x` project and create a new `v0.1.0` project in its place.
+
+```sh
+# Ensure SDK version is v0.1.0
+$ operator-sdk --version
+operator-sdk version 0.1.0
+
+# Create new project
+$ cd $GOPATH/src/github.com/example-inc/
+$ mv memcached-operator old-memcached-operator
+$ operator-sdk new memcached-operator
+$ ls
+memcached-operator old-memcached-operator
+
+# Copy over .git from old project
+$ cp -rf memcached-operator/.git old-memcached-operator/.git 
```

Comment:
This copies new to old, right? Don't you want this:
```
cp -rf old-memcached-operator/.git memcached-operator/.git
```

Reply:
Yes it should be the other way around, good catch.